oaknational/Oak-Web-Application
# feat: adds notification card to home page

## Description

- Adds notification card to home page
- Adds `transform` css util
- Adds `Bell` icon
- Adds a new drop shadow variant (which is used for the notification card)
- Makes some home page text sentence case
- Removes "Support your team" link (for now)
- Updates heading text and styles on homepage

**NB** clicking it doesn't currently go anywhere as don't have a blog post link yet

## Issue(s)

Fixes #493 
Progresses #386 

## How to test

1. Go to {cloud link}
3. You should see the notification
4. You should be able to tab to the notification
5. Screen reader should read "About the future of oak" when scrolling through links


## Screenshots

![image](https://user-images.githubusercontent.com/12934669/185975315-2c773563-0de0-4e8b-8b71-1bf18f8f5df4.png)

![image](https://user-images.githubusercontent.com/12934669/185975374-15b2da58-bf0f-4e50-8786-a15a5d5264f0.png)



## Checklist
